pkg/ip/link_linux.go

```diff
@@ -73,7 +77,7 @@ func makeVeth(name string, mtu int) (peerName string, veth netlink.Link, err err
 			return
 
 		case os.IsExist(err):
-			if peerExists(peerName) {
+			if peerExists(peerName) && vethPeerName == "" {
```

Comment:
Only when vethPeerName was empty, it would go with the current retry logic. @mars1024

Reply:
Looks good ~